include/xalloc.h

```diff
@@ -8,89 +8,94 @@
  */
 
 #ifndef UTIL_LINUX_XALLOC_H
-#define UTIL_LINUX_XALLOC_H
+# define UTIL_LINUX_XALLOC_H
 
-#include <stdlib.h>
-#include <string.h>
+# include <stdlib.h>
+# include <string.h>
 
-#include "c.h"
+# include "c.h"
 
-#ifndef XALLOC_EXIT_CODE
-# define XALLOC_EXIT_CODE EXIT_FAILURE
-#endif
+# ifndef XALLOC_EXIT_CODE
+#  define XALLOC_EXIT_CODE EXIT_FAILURE
+# endif
 
-static inline void __attribute__((__noreturn__))
-__err_oom(const char *file, unsigned int line)
+__attribute__((__noreturn__))
+static inline void __err_oom(const char *file, unsigned int line)
 {
```

Comment:
Why did you move __attribute__ before "static inline" and why for this function? We use it everywhere in the original way or at the end of the function declaration.

Reply:
Quite frankly documentation how to place function attributes in function declaration leaves lots of options open, so I took a punt and tried to make these easier to read. Now when before function placement is found unacceptable how about in between scope and return value type on each attribute on ones own line? Commit 9262473624c4220c550f4997fba1a57add20c6e3 does that.